org.opent2t.sample.binaryswitch.superpopular/com.wink.binaryswitch/js/thingTranslator.js

```diff
@@ -121,6 +123,8 @@ function resourceSchemaToProviderSchema(resourceId, resourceSchema) {
 
     if ('power' === resourceId) {
         desired_state['powered'] = resourceSchema.value;
+    }else{
+        throw new OpenT2TError(501, OpenT2TConstants.NotImplemented);

```

Comment:
Fixed.

---
In reply to: [106230145](https://github.com/openT2T/translators/pull/197#discussion_r106230145) [](ancestors = 106230145)